code/modules/clothing/modular_armor/attachments/modules.dm

```diff
@@ -99,6 +99,16 @@
 	icon_state = "mod_fire_som"
 	item_state = "mod_fire_som_a"
 
+/obj/item/armor_module/module/fire_proof/helmet
```

Comment:
Think it's better to have it be a child of main surt? Doesn't look like it inherits anything meaningful.

Reply:
Nah its actually bad to be a child of surt, since the hat alone will make you not combustable lol.